src/components/Sidebar/Sidebar.js

```diff
@@ -212,7 +212,6 @@ const Wrapper = animated(styled.nav.attrs({
     ${COLORS.blue[900]},
     ${COLORS.blue[700]}
   );
-  transform: translateX(${props => props.offset});
```

Comment:
Should this be in a separate PR to get the fix on master asap?

Reply:
A separate PR would be better so it's clear that we're having it on master.
I think it's a bit bad that I have to deal with it in every PR.

I'll create a quick PR so we can fix that on master. Then we can merge/rebase master here.